packages/knip/fixtures/plugin-ast/vite.config.ts

```diff
@@ -0,0 +1,17 @@
+import { defineConfig } from 'vite';
+import react from '@vitejs/plugin-react';
+import { TanStackRouterVite } from '@tanstack/router-plugin/vite';
+
+export default defineConfig({
+  plugins: [
+    TanStackRouterVite({
+      target: 'react',
+      autoCodeSplitting: true,
+      routesDirectory: './routes',
+      generatedRouteTree: './gen/routeTree.gen.ts',
+      // routeFileIgnorePrefix: '-',
+      // quoteStyle: 'single',
```

Comment:
I'm reviewing commit by commit. Maybe this was removed in a later commit?

Reply:
Nice catch, thanks